lxd/resource_lxd_instance_test.go

```diff
@@ -892,6 +906,21 @@ resource "lxd_instance" "instance1" {
 	`, name)
 }
 
+func testAccInstance_updateConfig3(name string) string {
+	return fmt.Sprintf(`
+resource "lxd_instance" "instance1" {
+  name = "%s"
+  image = "images:alpine/3.16"
+  profiles = ["default"]
+  config = {
+    "cloud-init.vendor-data" = "#cloud-config"
+    "user.dummy" = 5
+    "user.user-data" = "#cloud-config"
```

Comment:
Should we test with `cloud-init.user-data` instead since `user.user-data` is documented as legacy?